src/gui/mainwindow.cpp

```diff
@@ -316,6 +316,22 @@ MainWindow::MainWindow(QWidget *parent)
             activateWindow();
             raise();
         }
+#ifdef Q_OS_WIN
```

Comment:
I think it is generally better to have the window created on all platforms. Can you remove the ifdef?